### PR TITLE
LPD-31152 - Fix DataSetViewFilters tests

### PR DESCRIPTION
**Pull request by juanjofgliferay**
## Task / Technical task
[LPD-30197](https://liferay.atlassian.net/browse/LPD-30197) / [LPD-31152](https://liferay.atlassian.net/browse/LPD-31152)

### CX filters tests removed
- AssertNoClientExtensionAvailableMessage (removed -> included in CX filters test) 
- CanAssertNoDefaultFilters (ignored -> included in CX filters test)
- CanCancelClientExtensionFilterCreation (removed -> included in CX filters test)
- CanCloseClientExtensionFilterCreation (removed -> included in CX filters test)
- CanCreateClientExtensionFilter (removed -> included in CX filters test)
- CanNotCreateClientExtensionFilterWithClientExtensionFieldEmpty (removed -> included in CX filters test)
- CanNotCreateClientExtensionFilterWithFilterByFieldEmpty (removed -> included in CX filters test)
- CanCancelFilterCreationOnCancelButton (ignored -> included in CX filters test)
- CanCancelFilterCreationOnCloseButton (ignored -> included in CX filters test)
- CanCancelOnCancelButton (ignored -> duplicated included in CX filters test)
- CanCancelOnCloseButton (ignored -> duplicated included in CX filters test)
- AssertInUseLabelInClientExtensionFilterFilterByField (removed -> duplicated)
- CanAssertNoDefaultFilters (ignored -> duplicated included in Picklist filters test)
- TheNameFieldCannotBeEmpty (ignored -> duplicated included in CX filters test)

### Selection Filters tests removed
- SelectionTypeWithoutPreselectionValues (removed -> included in Picklist filters test)
- CanNotLeaveFilterByEmpty (ignored -> included in Picklist filters test)
- ConfirmPicklistIsNecessaryToCreateSelectionType (removed - included in Picklist filters test)
- CanAssertSourceOption (ignored -> included in Picklist filters test)

### Marked as Ignored tests removed (Not matching actual UI)
- AssertNameIsFilledByDefault
- CanAssertFieldsIsPresent
- CanAssertFilterByIsNotEditable
- CanAssertSourceOptionTooltip
- CanCreateFilters

### CODE REVIEW

**Comment by juanjofgliferay** [comment]
ci:test:frontend

**Comment by juanjofgliferay** [comment]
ci:test:playwright

**Comment by juanjofgliferay** [comment]
ci:test:sf

**Comment by juanjofgliferay** [comment]
ci:test:playwright

**Comment by juanjofgliferay** [comment]
ci:test:sf

**Comment by juanjofgliferay** [comment]
ci:test:playwright

File: modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/clientExtensionFilters.spec.ts [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+                filterBy: DATE_FIELD_NAME,
+                name: filterLabel,
+                clientExtension: clientExtensionName
+            });
+
+            await filtersPage.saveAddFilterModal();
+        });
+
+        await test.step('Check that the client extension filter is in the list', async () => {
+			await expect(
```
Comment (dsanz) [inline modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/clientExtensionFilters.spec.ts:167]: missing SF? there's an issue with pw SF . You can use this code https://github.com/liferay-frontend/liferay-portal/pull/4308 to fix it

Comment (juanjofgliferay) [reply modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/clientExtensionFilters.spec.ts:125]: Yep. I was waiting for this PR to be merged

**Comment by juanjofgliferay** [comment]
ci:test:playwright

**Comment by juanjofgliferay** [comment]
DSM CX Filters tests passing :tada: 
[Testray frontend-data-set-admin-web report](https://ff0c6ac131f3e99f0e29be842eeb583d1dc01b076d6bb3e2832458c-apidata.googleusercontent.com/download/storage/v1/b/testray-results/o/2024-07%2Ftest-1-30%2Ftest-portal-acceptance-pullrequest(master)%2F9630%2Fplaywright-js-tomcat90-mysql57-jdk8%2F51%2F0%2Fplaywright-report%2Findex.html?jk=AYBlUPBK4BMyW5rE5yrxGGmqVYL9Z7BGLzLgG6jtW6RDsrr40jzA794cLD0l2jE7bA3OSsVIqg7JctB3eNxKuF9AC03R0Iz1fblqTshWVa9JwA4ycZ4wcR9-3ypVMj8re_2yLYiPBQHmXmnK3jRruqWKGwHrH7C1jkjovB7IuEZj33u5YCObJhePG3s7jxXyi0xVXSVh9MEOHoznUQTufNTnbZduaLiiNR7Nhel3G2jqr8i-DwvMrk9VLyUCD4ciJKzIFh6stVIro9oI3BP0DHUPfHwqaibwpna8f4caSEdu8zMlYfwd6igq8hHaEbMDUcIAIYiQMmiXugVybeeqkOCCC-qcL9th4qonYvfS9Cbi4KFu609TPC-s04STXLQPiEQFeyFfne6DnJ4RE7mIaeHhJcSl-T-wA1NevpZPa76lePaabo_euDfUkRb9VfAzfi9rmHQ5z8kJtqT_Ofh956Rm7BlLG0QYqE0CkoDQI9zoboxGHqRLBeVD8Rg3FxFsBrNkgyxfx30HsppW3qnol0PaicbcS3zCQHQW5nKHwCk4hdgY6JIEkrYreZDYMRxJMdi45rNEv9FLeOo11JlFYxWoD3BcDfHXrwt9Xi8AYqZ0nStbZvGI5fm-7eTkDOXIQ-YbBv9NELiW4GqF0xMvx3KolY1_PeLqtzlAsInm-Zn1GHwA2KxfpYFfFMwAc6KF6MuRJe-a52USCFmCmzH2J7wGwDMvjI52VwTvSx5WnbAPwtc4mJhY8mMO4w5saUV2xg8f4eOfp_ho5R2nvHqyJKQ5_Jb-dmvuV_K6ffUc08owBgfHlF1iiNOBwq7RCa8Sw2wWi5Z_t4GAd-9bTGKHqKN-_X5Yt7bvwu55WZONRZMLZLm9O2RnKmOmpwwBXPWSwpfP7o88AgMbuM7uSXkqG6_j6S7Y5TsdivsCguHguZhiNHuMAvGuFyGoMLu8CGpa83mm-TRkp2cbe4JGyyqMl4-ssT_O-Z7d63mD6n4Wf5BwLC6PviZLpjjb96piu-XGRohe9wlm2a3qAu8MbwX0TgIf0SxGNnjiB1tGUAnLs-vYkkzJSFkCQV0823jCoEq2Dsy3eL7d4y7Pz-GNW_A4L6uoS5r1GxaH6tqWhgMmcgByA0UMp2Rc05L1jERUaUipLTrAmb2HOHdw1pg6WSg1CUVYJ5ZLr8RYKs72WMtMTho68wga7RiUVmx-AF6tWwFAtfUzC6J7QJkt1crJi_Hd50ELv2giMo8SR715cI3AzuLKsT1SqRB2oKMQ8gB2-aHOIL00uGGxRoKGTJlGvSQvTatxgqO62qThajxkaSPuo8t0lbc&isca=1#?q=p:frontend-data-set-admin-web)

**Comment by juanjofgliferay** [comment]
ci:forward

**Pull request closed without merging**